packages/dapp/env.d.ts

```diff
@@ -1,6 +1,11 @@
 /// <reference types="vite/client" />
 
 interface ImportMetaEnv {
+  readonly BASE_URL: string;
+  readonly DEV: boolean;
+  readonly MODE: 'development' | 'testnet' | 'production' | 'staging';
+  readonly PROD: boolean;
+  readonly SSR: boolean;
```

Comment:
This is not necessary, right? I am not sure about the type definitions on those env vars.
Does it even make sense to define the MODE variants here?